model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -147,16 +148,17 @@ case class Workspace(
 object Workspace {
   /** convenience constructor that defaults workspace version to v1 and google project to namespace */
   def apply(namespace: String,
-    name: String,
-    workspaceId: String,
-    bucketName: String,
-    workflowCollectionName: Option[String],
-    createdDate: DateTime,
-    lastModified: DateTime,
-    createdBy: String,
-    attributes: AttributeMap,
-    isLocked: Boolean = false): Workspace = {
-    new Workspace(namespace, name, workspaceId, bucketName, workflowCollectionName, createdDate, lastModified, createdBy, attributes, isLocked, WorkspaceVersions.V1, GoogleProjectId(namespace), None)
+
+           name: String,
+           workspaceId: String,
+           bucketName: String,
+           workflowCollectionName: Option[String],
+           createdDate: DateTime,
+           lastModified: DateTime,
+           createdBy: String,
+           attributes: AttributeMap,
+           isLocked: Boolean = false): Workspace = {
+    Workspace(namespace, name, workspaceId, bucketName, workflowCollectionName, createdDate, lastModified, createdBy, attributes, isLocked, WorkspaceVersions.V1, GoogleProjectId(namespace), None, None)
```

Comment:
is `new` not needed here?

Reply:
I believe it is effectively equivalent, but I did do a quick google check and there is a functional difference so I will add it back.